### ISSUE TITLE
rancher2_cluster: inconsistent final plan produced

### ISSUE THREAD

**Issue by airbjorn** [rancher/terraform-provider-rancher2#802]
It seems, that in some cases rancher2_cluster thinks there is something to do, although it isn't.

- We are using custom cacerts for authorization.
- Even without changes, calling `terraform plan` shows that there is something to do. The plan itself exits with 0.
- When calling `terraform apply` however, we get this error:
 ```
│ Error: Provider produced inconsistent final plan
│
│ When expanding the plan for local_file.kubeconfig_cluster to include new values learned so far during apply, provider
│ "registry.terraform.io/hashicorp/local" produced an invalid new value for .content: inconsistent values for sensitive attribute.
│
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```
- trace log see [tf-trace.log](https://github.com/rancher/terraform-provider-rancher2/files/7553997/tf-trace.log)
- definitions of the Ressources "cluster" and "local_file" see [cluster-local_file.tf](https://github.com/rancher/terraform-provider-rancher2/files/7554002/cluster-local_file.log)
- Workaround / Quick Fix is shown in the file [cluster-local_file.diff](https://github.com/rancher/terraform-provider-rancher2/files/7561889/cluster-local_file.diff.txt)
- used versions:
Rancher 2.6.2
```
Terraform v1.0.11
on linux_amd64
+ provider registry.terraform.io/gavinbunney/kubectl v1.13.1
+ provider registry.terraform.io/hashicorp/dns v3.2.1
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
+ provider registry.terraform.io/rancher/rancher2 v1.21.0
+ provider registry.terraform.io/rancher/rke v1.2.4
```

**Comment by rawmind0** [comment]
Related with issue https://github.com/rancher/terraform-provider-rancher2/issues/789

**Comment by rawmind0** [comment]
Submitted PR https://github.com/rancher/terraform-provider-rancher2/pull/837 to fix the issue

**Comment by rawmind0** [comment]
The fix has been included on new release v1.22.2. Please, reopen issue if needed
